helm/templates/service.yaml

```diff
@@ -10,6 +10,6 @@ spec:
     - port: {{ .Values.service.port }}
       targetPort: grpc-port
       protocol: TCP
-      name: grpc-port
+      name: {{ .Chart.Name }}
```

Comment:
why are we doing this @ravisingal ?

Reply:
same question :)

Reply:
istio can automatically detect http and http/2 traffic. with manual protocol detection, client was complaining about protocol mismatch between grpc server and client (server using ssl and client was using plaintext).

Reply:
I thought the issue you were having with mTLS between some services was that the protocol sniffing was causing issues, so we were going to go back and use manual protocol specification.

Can you explain the protocol mismatch a little more.   The client and server are both grpc right?  Why is there a mismatch between ssl and plaintext?